src/test/java/FileUtilityTest.java

```diff
@@ -3,26 +3,40 @@
 import org.junit.jupiter.api.Assertions;
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
 import utilities.FileUtility;
 import utilities.PropertyUtility;
-
+import utilities.ShellUtility;
 import java.io.IOException;
 import java.nio.file.*;
+import java.nio.file.attribute.FileTime;
 import java.nio.file.attribute.PosixFilePermission;
+import java.time.Duration;
+import java.time.Instant;
 import java.util.*;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.concurrent.TimeoutException;
+import java.util.function.Function;
 import java.util.stream.Collectors;
 import java.util.stream.IntStream;
 
 class FileUtilityTest {
 
     private Logger logger = LogManager.getLogger(FileUtilityTest.class);
-    final Path testBedPath;
+    private static int thread_sleep = Integer.parseInt(
```

Comment:
What is this sleep for? What is the unit? Please rename it to contain that info.